.env

```
@@ -21,13 +21,13 @@ _APP_OPTIONS_ROUTER_PROTECTION=disabled
 _APP_OPTIONS_FORCE_HTTPS=disabled
 _APP_OPTIONS_ROUTER_FORCE_HTTPS=disabled
 _APP_OPENSSL_KEY_V1=your-secret-key
-_APP_DNS=8.8.8.8
+_APP_DNS=172.16.238.100 # CoreDNS
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The _APP_DNS key should go before the _APP_EDITION key

(UnorderedKey)